charts/base-services/templates/rules/general.yaml

```
@@ -9,11 +9,18 @@ spec:
   groups:
   - name: general.rules
     rules:
-    - alert: TargetDown
+    - alert: TargetDown short
       annotations:
         message: '{{`{{ $value }}`}}% of the {{`{{ $labels.job }}`}} targets are down.'
       expr: 100 * (count(up == 0) BY (job) / count(up) BY (job)) > 10
       for: 10m
       labels:
         severity: warning
+    - alert: TargetDown long
+      annotations:
+        message: '{{`{{ $value }}`}}% of the {{`{{ $labels.job }}`}} targets are down.'
+      expr: 100 * (count(up == 0) BY (job) / count(up) BY (job)) > 10
+      for: 30m
```

Also maybe 30m is too much (we could lose heartbeat on validators)
```suggestion
      for: 15m
```